Add a simple command API (#1680)
f3d-app/f3d: doc/libf3d/OPTIONS.md

    std::cout << "Line Width: unset" << std::endl;
  }
```

> It even more true with the few optional boolean options as std::optional has a boolean operator.

Reviewer: ```suggestion
It's even more true with the few optional boolean options as std::optional has an implicit boolean cast operator.
```
